templates/todo/projects/nodejs-mongo/.repo/terraform/infra/main.tfvars.json

```diff
@@ -1,5 +1,6 @@
 {
     "location": "${AZURE_LOCATION}",
     "environment_name": "${AZURE_ENV_NAME}",
-    "principal_id": "${AZURE_PRINCIPAL_ID}"
+    "principal_id": "${AZURE_PRINCIPAL_ID}",
+    "useAPIM" : "${USE_APIM=false}"

```

Comment:
Who is processing this string interpolation? Is that terraform itself and it already supports specifying default values like we see here?